creator-node/sequelize/migrations/20220427014326-add_playlist_to_clockrecords_enum.js

```diff
@@ -0,0 +1,129 @@
+'use strict'
+
+/**
+ * New table - Playlist
+ * CNodeUsers Table considered a Reference Table only
+ */
+
+const tableName = 'Playlists'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    console.log('STARTING MIGRATION 20220427014326-add_playlist_to_clockrecords_enum')
+    const transaction = await queryInterface.sequelize.transaction()
+    
+    // Create Playlist table
+    await createPlaylistTable(queryInterface, Sequelize, transaction)
+
+    await addPlaylistToClockRecordSourceTables(queryInterface, Sequelize, transaction)
+    
+    // Add composite unique constraint on (playlistId, clock) to Playlist
+    await addCompositeUniqueConstraints(queryInterface, Sequelize, transaction)
```

Comment:
i don't think this has to be done in a separate step from the createPlaylistTable - i know you modeled it off the previous migrations but they necessarily separate it since the data tables were created before the clockRecord table and associations were
for this migration the `createTable()` query should handle these constraints as well

Reply:
lets chat offline - i tried to combine these together but was unsuccessful